examples/models/sklearn_spacy_text/sklearn_spacy_text_classifier_example.ipynb

```diff
@@ -11,6 +11,7 @@
     "For this, we will be using SpaCy for the word tokenization and lemmatization. \n",
```

Comment:
<p>It would be good to remove the outputs before merging the PR.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/3076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>